components/ui/Headings.jsx

```diff
@@ -0,0 +1,35 @@
+export const H1 = ({ text, styles }) => {
+    return (
+        <h1 className={`lg:text-23xl lg:leading-15.332  ${styles}`}>{text}</h1>
+    )
+}
+
+export const H2 = ({ text, styles }) => {
```

Comment:
remove the unnecessary spaces inside the props

Reply:
Applied